scripts/seal.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (robot) ->
+  robot.hear /seal me up/i, (res) ->
```

Comment:
Hi! On second thought, I think the cases where someone might want to trigger the seal while adding a word or character to the incantation, ie. “seal me up!” or “seal me up, please” are more likely than them saying “seal me up” in a conversation without wanting to summon the seal, so I’d rather leave it like this for now and monitor how it goes, to see if making it more restrictive is necessary.